en/flight_controller/omnibus_f4_sd.md

```diff
@@ -135,27 +135,41 @@ Here is an example implementation. I used a Spektrum plug to get 3.3v from the D
 ![Omnibus F4 SD Pullup Implementation](../../assets/flight_controller/omnibus_f4_sd/pullup.jpg)
 
 
-## CRSF Crossfire Telemetry
-In addition to the [FrSky Telemetry](../peripherals/frsky_telemetry.md), the Omnibus also supports TBS CRSF Crossfire telemetry.
-It is used to send telemetry data from the flight controller (the vehicles's attitude, battery, flight mode and GPS data) to the RC transmitter (Taranis).
+## RC Telemetry
 
-Benefits are:
+The Omnibus supports telemetry to the RC Transmitter using [FrSky Telemetry](../peripherals/frsky_telemetry.md) or [CRSF Crossfire Telemetry](#crsf_telemetry).
+
+
+### CRSF Crossfire Telemetry {#crsf_telemetry}
+
+TBS CRSF Crossfire telemetry is used to send telemetry data from the flight controller (the vehicle's attitude, battery, flight mode and GPS data) to the RC transmitter (Taranis).
+
+Benefits over FrSky telemetry include:
 - Only a single UART is needed for RC and telemetry.
 - The CRSF protocol is optimized for low latency.
 - 150 Hz RC update rate.
 - The signals are uninverted and thus no (external) inverter logic is required.
 
-It is best to use the [TBS Crossfire Nano RX](http://team-blacksheep.com/products/prod:crossfire_nano_rx), since it is specifically designed for small Quads
-(the referenced link also contains the documentation).
+For Omnibus we recommend the [TBS Crossfire Nano RX](http://team-blacksheep.com/products/prod:crossfire_nano_rx), since it is specifically designed for small Quads.
+
+On the handheld controller (e.g. Taranis) you will also need a [Transmitter Module](http://team-blacksheep.com/shop/cat:rc_transmitters#product_listing).
+This can be plugged into the back of the RC controller.
+
+> **Note** The referenced links above contains the documentation for the TX/RX modules.
+
+#### Setup
+
+Connect the Nano RX and Omnibus pins as shown:
+
+Omnibus UART1 | Nano RX
```

Comment:
FYI This is just a layout choice. 

@bkueng I was hoping here to link to information about the UART1 pins. However if you look above, we don't say what the pinout is (we do cover UART6). So I suggest that perhaps you add information on the pinout for UART1 above and link to it.

Reply:
> So I suggest that perhaps you add information on the pinout for UART1 above and link to it.

I don't know what more there could be added. UART1 is already mentioned for RC and the pictures contain the pins as well. Note that on that board you directly solder pins, there are no connectors (it's a different world than Pixhawk's).

Reply:
Will your audience know that TX1/RX1 are the TX/RX for the UART1? I'm more used to seeing UARTs labeled like "GPS", "TELEM1". If so, then fine! Often UARTs have other pins - for CTS etc. Looks like some UARTs do here and some don't -s that worth noting?